target/linux/ath79/image/generic.mk

```diff
@@ -22,6 +22,20 @@ define Device/buffalo_wzr-hp-g450h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-g450h
 
+define Device/dlink_dir-825-b1
+  ATH_SOC := ar7161
+  DEVICE_TITLE := D-LINK DIR-825 B1
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma
```

Comment:
it is default in https://github.com/openwrt/openwrt/blob/2f98868d9bad71da15cd4101d0ae293801b0e73f/target/linux/ath79/image/Makefile and can be dropped here